### ISSUE TITLE
Why the two stop/start scripts?

### ISSUE THREAD

**Issue by orweinberger** [jonklinger/Raspbian-Bitnote-Installer#1]
Why have two scripts that checks if bitcoind is running instead of one?

You could just do something like
`@hourly /home/pi/startcoin.sh`

**Comment by jonklinger** [comment]
one is meant to restart if bitcoind is down, the other to kill it.

**Comment by orweinberger** [comment]
But the script checks if bitcoind is running, if not - it runs it. Why do you need to kill it?

**Comment by jonklinger** [comment]
because it might become a zombie process, or hang. 

Killing it makes sense when you need the initial sync. Maybe afterwards all is ok.

**Comment by jonklinger** [comment]
(sorry for closing)

**Comment by orweinberger** [comment]
This is the bash script I use to make sure bitcoind is running on my fullnode.co nodes, never had to kill nodes, not on initial sync and not afterwards. Feel free to use it of course:

```
#! /bin/bash

case "$(pidof bitcoind | wc -w)" in

0)  echo "Restarting Bitcoind:     $(date)" >> /home/fullnode/bitcoind.txt
    /home/fullnode/bitcoin-0.9.3-linux/bin/64/bitcoind -server &
    ;;
1)  # all ok
    ;;
*)  echo "Removed double Bitcoind: $(date)" >> /home/fullnode/bitcoind.txt
    kill $(pidof bitcoind | awk '{print $1}')
    ;;
esac
```

**Comment by jonklinger** [comment]
yeah, but then in the case where I am running "bitcoind getinfo" to log the blocks to the log file, it will kill both processes.

**Comment by orweinberger** [comment]
No it won't. Add your getinfo command instead of the `# all ok` part.

So the crontab runs every 5 minutes let's say, initial run falls into case 0 (since it wasn't running before). 

5 Minutes passes, the script runs again, let's say everything is ok and bitcoind is running, so case 1 will apply which will do the `getinfo`. So as long as everything is ok, every X minutes you'll get the getinfo logged to a file, whenever the process is not running, it will run it. And whenever there is more than one instance of bitcoind, it will kill one of the processes.
